lib/centralisation/providers/openLink.dart

```diff
@@ -0,0 +1,9 @@
+import 'package:url_launcher/url_launcher.dart';
+
+void openLink(String url) async {
+  if (await canLaunch(url)) {
+    await launch(url);
```

Comment:
Take care, both canLauch and launch are deprecated, you can use canLauchUrl and launchUrl instead